command/maintenance_window_create.go

```diff
@@ -59,7 +61,7 @@ func (c *MaintenanceWindowCreate) Run(args []string) int {
 		return -1
 	}
 	log.Debugf("%#v", m)
-	if _, err := client.CreateMaintenanceWindows(m); err != nil {
+	if _, err := client.CreateMaintenanceWindowWithContext(context.Background(), "", m); err != nil {
```

Comment:
Could you undo this change? Since we're not actually making use of the `context` value here, there isn't a reason to update this to call the new method. I'm okay with the new code making use of the new methods. :+1: